src/ShadowDOM/wrappers/Range.js

```diff
@@ -24,6 +24,13 @@
   var ShadowRoot = scope.wrappers.ShadowRoot;
 
   function getHost(node) {
+    var parent = node;
+    while (parent = parent.parentNode) {
+      if (parent.shadowRoot) {
```

Comment:
This is for consistency with code at the end of the function.
Function is expected to get Host, but if there is no suitable host, then `null` is returned (which below in `getHost(node) || node` effectively results in node itself).